src/TestableHttpClient/Utils/Value.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace TestableHttpClient.Utils;
+
+internal abstract record Value
```

Comment:
Custom tostring and equals methods, as well as debugging attributes might be useful here.

Reply:
Will not do, since this implies creating a GetHashCode method as well, to much hassle...